test/com/google/enterprise/gsafeed/RecordTest.java

```diff
@@ -519,6 +524,24 @@ public void testLastModifiedDifferentFormatWithValidation()
     new GsafeedHelper().unmarshalWithDtd(feed);
   }
 
+  // Add an even more straightforward date format test here.
```

Comment:
Is this a TODO(aptls)?

Reply:
No, just a poorly-phrased comment. I wanted to include a basic check of the formatting with no extra layers.